ppgan/engine/trainer.py

```diff
@@ -47,13 +48,15 @@ def __init__(self, cfg):
             self.distributed_data_parallel()
 
         self.logger = logging.getLogger(__name__)
+        self.vdl_logger = visualdl.LogWriter(logdir=cfg.output_dir)
```

Comment:
Add a bool flag to determine whether use visualdl.

Reply:
ok